blocks/tabs/tabs.js

```diff
@@ -1,62 +1,81 @@
+
+      const fragmentBlock = await loadFragment(fragmentPath);
+      if (fragmentBlock) {
+        const lastChild = tabpanel.lastElementChild;
+        const fragmentChild = fragmentBlock.querySelector('.section');
+        if (fragmentChild) {
+          lastChild.innerHTML = fragmentChild.innerHTML;
+        }
       }
-      return;
-    }
-    const id = toClassName(tab.textContent);
-
-    // decorate tabpanel
-    const tabpanel = block.children[i];
-    tabpanel.className = 'tabs-panel';
-    tabpanel.id = `tabpanel-${id}`;
-    tabpanel.setAttribute('aria-hidden', !!i);
-    tabpanel.setAttribute('aria-labelledby', `tab-${id}`);
-    tabpanel.setAttribute('role', 'tabpanel');
-    if (!hasWrapper(tabpanel.lastElementChild)) {
-      tabpanel.lastElementChild.innerHTML = `<p>${tabpanel.lastElementChild.innerHTML}</p>`;
     }
+  }
+  tabpanel.className = 'tabs-panel';
+  tabpanel.id = `tabpanel-${id}`;
+  tabpanel.setAttribute('aria-labelledby', `tab-${id}`);
+  tabpanel.setAttribute('role', 'tabpanel');
+  tabpanel.setAttribute('aria-hidden', index !== 1);
+  if (!hasWrapper(tabpanel.lastElementChild)) {
+    tabpanel.lastElementChild.innerHTML = `<p>${tabpanel.lastElementChild.innerHTML}</p>`;
+  }
 
-    // build tab button
-    const button = document.createElement('button');
-    button.className = 'tabs-tab';
-    button.id = `tab-${id}`;
-    button.innerHTML = tab.innerText;
-    button.setAttribute('aria-controls', `tabpanel-${id}`);
-    button.setAttribute('aria-selected', !i);
-    button.setAttribute('role', 'tab');
-    button.setAttribute('type', 'button');
-
-    //  Click events
-    button.addEventListener('click', () => {
-      block.querySelectorAll('[role=tabpanel]').forEach((panel) => {
-        panel.setAttribute('aria-hidden', true);
-      });
-      tablist.querySelectorAll('button').forEach((btn) => {
-        btn.setAttribute('aria-selected', false);
-      });
-      tabpanel.setAttribute('aria-hidden', false);
-      button.setAttribute('aria-selected', true);
+  const button = document.createElement('button');
+  button.className = 'tabs-tab';
+  button.id = `tab-${id}`;
+  button.innerHTML = tab.innerText;
+  button.setAttribute('aria-controls', `tabpanel-${id}`);
+  button.setAttribute('aria-selected', index === 1);
+  button.setAttribute('role', 'tab');
+  button.setAttribute('type', 'button');
+
+  button.addEventListener('click', () => {
+    block.querySelectorAll('[role=tabpanel]').forEach((panel) => {
+      panel.setAttribute('aria-hidden', true);
+    });
+    tablist.querySelectorAll('button').forEach((btn) => {
+      btn.setAttribute('aria-selected', false);
     });
-    tablist.append(button);
-    tab.remove();
+    tabpanel.setAttribute('aria-hidden', false);
+    button.setAttribute('aria-selected', true);
   });
+  tablist.append(button);
+  tab.remove();
+}
+
+export default async function decorate(block) {
+  const tablist = document.createElement('div');
```

Comment:
Consider using `nav` instead of div for the `tablist` element to provide better structure to search engines.

`const tablist = document.createElement('nav');`

Reply:
@amogh-daryapurkar - This should be `div` only. Since the Tab doesn't have any navigation. We should use the `nav` element only for navigation. Otherwise, this will cause an issue in the screen reader.